src/Features/Renderer.cpp

```diff
@@ -948,11 +952,23 @@ static bool SND_RecordBuffer_Hook(void)
 void Renderer::Frame()
 {
     if (Renderer::isDemoLoading && engine->hoststate->m_currentState == HS_RUN) {
-        Renderer::isDemoLoading = false;
         if (!g_render.isRendering.load() && sar_render_autostart.GetBool()) {
+
+            if (sar_render_autostart_odd_tick.GetBool()) {
+                // If the parity of the current tick does not match the parity of the first non-zero demo packet, don't start rendering
+                if (engine->demoplayer->GetTick() % 2 != Renderer::demoParity) {
+                    engine->SendToCommandBuffer("sv_alternateticks 0", 0);
+                    return;
+                } else {
+                    engine->SendToCommandBuffer("sv_alternateticks 1", 0);
```

Comment:
Probably cleaner to do `sv_alternateticks.SetInt(1)` here. I'm actually surprised this works with the second arg being 0, I thought that delayed it by a tick or so, but I guess not (same applies to the one above)